README.md

```diff
@@ -62,6 +65,7 @@ files:
   - roadmap.pdf
 
 You can verify that the API has launched correctly by navigating to the following urls via any web browser.
+
 - http://localhost:9090/api/v1/Solutions/health/live
 - http://localhost:9090/api/v1/Solutions/health/ready
```

Comment:
We should enclose bare URLs in angled brackets (without the brackets some parsers won't convert the URL to a link).

Example:
`<http://localhost:9090/api/v1/Solutions/health/live>`

Alternatively, if we don't want a link we should enclose each URL in a code block.